src/components/shared/Table.js

```diff
@@ -1,42 +1,105 @@
 import React from "react"
-import MaterialTable from "material-table"
+import MUIDatatable from "mui-datatables"
 import Typography from "@material-ui/core/Typography"
+import useMediaQuery from "@material-ui/core/useMediaQuery"
+import { createMuiTheme, MuiThemeProvider } from "@material-ui/core/styles"
 
 export default function Table({ title, data, columns, marginTop, grouping }) {
```

Comment:
This is not going to stop me from merging this, but we're not using `grouping` anymore, so can remove it here and from `StatPage` and `abilities`.

Reply:
Yeah, I thought of that, but like I mentioned either here or on the issue, I think it should be possible to do some type of grouping (probably similar to how I did it on my site), so I figured I'd just leave it as a placeholder for now.

Reply:
Great, let's leave it then, thanks.